src/css/components/_title-bar.scss

```diff
@@ -0,0 +1,41 @@
+.vjs-title-bar {
+
+  // At a base inherited font-size of 10px, the title bar overall height should
+  // be 96px with the area of text occupying the first 48px and the rest being
+  // padding. This leaves plenty of room for the gradient to fade to
+  // transparent while maintaining an WCAG AA-compliant contrast ratio (tested
+  // using the TPGi Color Contrast Analyzer application) even on top of a solid
+  // white background.
+  @include linear-gradient(
```

Comment:
Confirmed that worst-case scenario is still above the WCAG AA threshold of 4.5:1 contrast ratio.

Reply:
Nice!